zebrad/tests/acceptance.rs

```diff
@@ -4,195 +4,292 @@
 #![warn(warnings, missing_docs, trivial_casts, unused_qualifications)]
 #![forbid(unsafe_code)]
 
-use abscissa_core::testing::prelude::*;
 use color_eyre::eyre::Result;
-use once_cell::sync::Lazy;
 use std::time::Duration;
 use zebra_test::prelude::*;
 
-/// Executes zebrad binary via `cargo run`.
-pub static RUNNER: Lazy<CmdRunner> = Lazy::new(CmdRunner::default);
+// Todo: The following 3 helper functions can probably be abstracted into one
```

Comment:
I think the only way to effectively deduplicate these would be to use a macro

Reply:
We could move this code into a utility function:

            .stdout(Stdio::piped())
             .stderr(Stdio::piped())
             .spawn2()
             .unwrap(),